switchmap/poller/snmp/snmp_manager.py

```
@@ -477,466 +504,468 @@ def query(
+            mp_model = 0 if auth.version == 1 else 1
+            auth_data = CommunityData(auth.community, mpModel=mp_model)
 
-        # Return
-        return result
+        return auth_data, transport_target
 
+    async def _do_async_get(
+        self, oid, auth_data, transport_target, context_data
+    ):
+        """Pure async SNMP GET using pysnmp."""
+        error_indication, error_status, error_index, var_binds = await getCmd(
```

Ruff (0.13.2)

Unpacked variable `error_index` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)